Buildings/Applications/DHC/EnergyTransferStations/CoolingDirectControlledReturn.mo

```diff
@@ -0,0 +1,396 @@
+within Buildings.Applications.DHC.EnergyTransferStations;
+model CoolingDirectControlledReturn
+  "Direct cooling ETS model for district energy systems with in-building 
+    pumping and controlled district return temperature"
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+    redeclare final package Medium1 = Medium,
+    redeclare final package Medium2 = Medium,
+    final m1_flow_nominal = mDis_flow_nominal,
```

Comment:
I am okay following your suggestion. I had configured the ports in the way that I had because the MBL standard is for fluid to flow from a1 to b1, and with the direct cooling ETS, the fluid flows from the district supply directly into the building supply. However for consistency with the rest of the ETS models, I will revise to have the fluid flow as shown here (flow paths typical for uncontrolled and controlled direct ETS models). If you have any other comments, please let me know.

![image](https://user-images.githubusercontent.com/44682485/87855217-045c4800-c8d4-11ea-8abd-9432e4ae7eda.png)